ts/features/newWallet/components/WalletCardsContainer.tsx

```diff
@@ -63,40 +71,56 @@ const ItwCardsContainer = ({
   const isItwValid = useIOSelector(itwLifecycleIsValidSelector);
   const isItwEnabled = useIOSelector(isItwEnabledSelector);
 
+  const eidInfoBottomSheet = useIOBottomSheetAutoresizableModal(
+    {
+      title: <ItwEidInfoBottomSheetTitle />,
+      component: <ItwEidInfoBottomSheetContent />
+    },
+    EID_INFO_BOTTOM_PADDING
+  );
+
   if (!isItwTrialEnabled || !isItwEnabled) {
     return null;
   }
 
-  const endElement: ListItemHeader["endElement"] = isItwValid
-    ? {
-        type: "badge",
+  const getHeader = (): ListItemHeader | undefined => {
+    if (!isItwValid) {
+      return undefined;
+    }
+    return {
+      iconName: "legalValue",
+      iconColor: "blueIO-500",
+      label: I18n.t("features.wallet.cards.categories.itw"),
+      endElement: {
+        type: "buttonLink",
         componentProps: {
-          text: I18n.t("features.itWallet.wallet.active"),
-          variant: "blue",
+          label: I18n.t(
+            "features.itWallet.presentation.bottomSheets.eidInfo.triggerLabel"
+          ),
+          onPress: eidInfoBottomSheet.present,
           testID: "walletCardsCategoryItwActiveBadgeTestID"
         }
       }
-    : {
-        type: "badge",
-        componentProps: {
-          text: I18n.t("features.itWallet.wallet.inactive"),
-          variant: "default",
-          testID: "walletCardsCategoryItwInactiveBadgeTestID"
-        }
-      };
+    };
+  };
 
   return (
-    <WalletCardsCategoryContainer
-      key={`cards_category_itw`}
-      testID={`walletCardsCategoryTestID_itw`}
-      cards={cards}
-      isStacked={isStacked}
-      header={{
-        label: I18n.t("features.wallet.cards.categories.itw"),
-        endElement
-      }}
-      footer={<ItwDiscoveryBanner ignoreMargins={true} />}
-    />
+    <>
+      <WalletCardsCategoryContainer
+        key={`cards_category_itw`}
+        testID={`walletCardsCategoryTestID_itw`}
+        cards={cards}
+        isStacked={isStacked}
+        header={getHeader()}
+        footer={
+          <>
+            <ItwDiscoveryBanner ignoreMargins={true} closable={false} />
+            <ItwWalletReadyBanner />
+          </>
```

Comment:
The `footer` prop was added because we needed to display "something" below the cards. But now the banners should appear above them.
What do you think about changing this prop from `footer` to `somethingelse` and move the content above the cards?

Reply:
Addressed in c480b3f5e58aa4f3e39b0240b776398ea7b070a1.